sway/commands/clipboard.c

```diff
@@ -0,0 +1,38 @@
+#include <wlc/wlc.h>
+#include <unistd.h>
+#include <string.h>
+#include "sway/commands.h"
+#include "stringop.h"
+
+static void send_clipboard(void *data, const char *type, int fd) {
+	if ((strcmp(type, "text/plain") != 0) &&
+			(strcmp(type, "text/plain;charset=utf-8") != 0)) {
```

Comment:
Sorry, more style nits. You don't need these extra parens and you should put `&&` on the next line.